dll/win32/ntdll_vista/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ list(APPEND SOURCE
     DllMain.c
     condvar.c
     srw.c
+    ${REACTOS_SOURCE_DIR}/sdk/lib/rtl/runonce.c
```

Comment:
Can we just link ntdll_vista with the `rtl` library to make this simpler?
Alternatively, `runonce.c` could just live here like the other files do, instead of being part of rtl?

Reply:
You can't link rtl, since ntdll_vista already links to ntdll, which will create a mess. But this will soon be exported by ntdll (export will be hidden unless the appcompat setting specifies vista, but our own dlls will be able to link to it secretly).